incydr/_jsonutils.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
```

Comment:
vendoring this instead of adding all of boltons as a dependency for now, although it might be worthwhile to bring in the whole thing, as some of the other utils might be useful...

https://boltons.readthedocs.io/en/latest/jsonutils.html